libraries/commerce/cart/helpers/shipping.js

```diff
@@ -0,0 +1,27 @@
+import appConfig from '@shopgate/pwa-common/helpers/config';
+
+/**
+ * Retrieves the shipping config based on the legacy configuration and the new Engage config.
+ * @returns {Object} The merged shipping configuration.
+ */
+export function getShippingConfig() {
+  const {
+    shippingHideLegacy = null, shippingUnavailableLegacy = null, shipping,
+  } = appConfig;
+  const {
+    show, freeShipping, unavailable, hint,
+  } = shipping;
+
+  // It is using the NMA config over the legacy config.
+  const showShipping = (typeof show !== 'undefined') ? show : shippingHideLegacy;
```

Comment:
`show` is `null` by default app config. This condition will be always true (by default)